spec/models/spotlight/resources/purl_spec.rb

```diff
@@ -1,15 +1,18 @@
 require 'spec_helper'
 
 describe Spotlight::Resources::Purl do
-  
   let :exhibit do
     double(solr_data: { }, blacklight_config: Blacklight::Configuration.new)
   end
+  let :blacklight_solr do
+    double
+  end
 
   subject { Spotlight::Resources::Purl.new url: "http://purl.stanford.edu/xf680rd3068" }
 
   before do
     allow(subject).to receive(:exhibit).and_return(exhibit)
+    allow(subject).to receive(:blacklight_solr).and_return(blacklight_solr)
```

Comment:
I think this could just return double;  splitting exhibit out above makes it easier to see the additional config added to the double.  Don't have strong feelings about it at this time.